src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_decorator.py

```diff
@@ -4179,6 +4179,7 @@ def test_construct_default_mc_profile(self):
                 "assign_kubelet_identity": None,
                 "enable_ultra_ssd": False,
                 "edge_zone": None,
+                "disable_local_accounts": False,
```

Comment:
I would appreciate it if you could also add some cases to test the getter function (I mean `get_disable_local_accounts`, through it quite straightforward for now) under class `AKSCreateContextTestCase`.